frontend/public/config/config.js

```diff
@@ -1,5 +1,6 @@
 export const config = {
   api_base: 'http://localhost:8000/api',
+  tfrs_base: 'http://localhost:3001',
```

Comment:
in local we're using port 3000

Reply:
I set this to 3001 as LCFS also runs on 3000 so if you were to run both locally at the same time TFRS would need to be on something else. Should I set them to the same?

Reply:
I do like the idea but there are quite a few places we would have to update this. vite config, cypress config, application settings, and a few more spots. Maybe leave it as 3000 and then adjust locally when testing this hybrid code. If you want to go through and update it everywhere im not opposed but we would have to test thoroughly.

Reply:
Updated to 3000.